husarion_ugv_crsf_teleop/setup.py

```diff
@@ -25,7 +25,8 @@
     packages=find_packages(exclude=["test"]),
     data_files=[
         ("share/ament_index/resource_index/packages", ["resource/" + package_name]),
-        (os.path.join("share", package_name, "config"), ["config/crsf_teleop.yaml"]),
+        (os.path.join("share", package_name, "config"), ["config/crsf_teleop_panther.yaml"]),
+        (os.path.join("share", package_name, "config"), ["config/crsf_teleop_lynx.yaml"]),
```

Comment:
Should work the same
```suggestion
        (os.path.join("share", package_name, "config"), glob(os.path.join("config", "*.yaml"))),
```